tests/expectations/cfg.compat.c

```diff
@@ -95,6 +95,11 @@ typedef struct {
 #endif
   ;
 } ConditionalField;
+#define ConditionalField_ZERO (ConditionalField){ 
+#if defined(X11)
```

Comment:
@emilio Bad news here. This implementation does not cover C. Any advice to only enable the feature for C++ with sufficient features?